templates/registration/signup.html

```diff
@@ -0,0 +1,84 @@
+{% extends '_base.html' %}
+
+{% block content %}
+
+
+<div class="container px-4 px-lg-5 h-100">
+    <div class="row h-100 align-items-center justify-content-center text-center">
+        <div class="col-lg-4 align-self-end">
+            <h1 class="m-5">Report Hub</h1>
+            {% if form.errors %}
+            {{ form.errors }}
+            {% endif %}
+
+            {% if messages %}
+                {% for message in messages %}
+                    {% if message.tags %}
+                            <div class="alert alert-{{ message.tags }}" role="alert">{{ message }}</div>
+                    {% endif %}
+                {% endfor %}
+            {% endif %}
+            <p>Humanitarian Decision Support in Real-Time</p>
+            <p>Create your account</p>
+            <form method="post" action="{% url 'register' %}">
+                {% csrf_token %}
+                <div class="mb-3">
+                    <input type="text" name="{{ form.name.html_name }}" id="{{ form.name.id_for_label }}" class="form-control" placeholder="Full Name" required>
+                </div>
+                <div class="mb-3">
+                    <input type="text" name="{{ form.username.html_name }}" id="{{ form.username.id_for_label }}" class="form-control" placeholder="Username" required/>
+                </div>
+                <div class="mb-3">
+                    <input type="text" name="{{ form.email.html_name }}" id="{{ form.email.id_for_label }}" class="form-control" placeholder="Email" required/>
+                </div>
+                <div class="mb-3">
+                    <input type="password" class="form-control" name="{{ form.password1.html_name }}" id="{{ form.password1.id_for_label }}" placeholder="Password" required/>
+                </div>
+                <div class="mb-3">
+                    <input type="password" class="form-control" name="{{ form.password2.html_name }}" id="{{ form.password2.id_for_label }}" placeholder="Confirm Password" required/>
+                    <span id='message'></span>
+                </div>
+                {% comment %} TODO: Add the other custom user details input here in future {% endcomment %}
+                <div class="mb-3 d-grid">
+                    <input type="submit" class="btn btn-outline-dark" value="Sign Up"/>
+                </div>
+            </form>
+            <p class="mb-0">Forgot your passowrd?</p>
+            <p class="mb-0">
+                Already have an account?
+                <a href="{% url 'login' %}">Login</a>
+            </p>
+        </div>
+    </div>
+</div>
+
+{% comment %} <script>
+    $('#id_password2').keyup(function(){
+        var pass1 = $('#id_password1');
+        var pass2 = $('#id_password2');
+        if(pass1.val() != pass2.val()){
+            debugger
+            pass1.attr({style:'box-shadow: 0 0 5px rgb(225 84 84)'});
```

Comment:
if we can extract these styles into CSS file and use django compressor workflow that would be great.